src/App.tsx

```diff
@@ -17,18 +50,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filteringBy={filteringBy}
+                ChangeFiltering={setFilteringBy}
+                query={query}
+                ChangeQuery={setQuery}
```

Comment:
Type attributes from small letter
```suggestion
              <TodoFilter
                filteringBy={filteringBy}
                changeFiltering={setFilteringBy}
                query={query}
                changeQuery={setQuery}
```